runbotics/runbotics-scheduler/.env.development

```diff
@@ -13,6 +13,7 @@ RUNBOTICS_SECRET='ODQ4ZDBjNTgzYTE3ZjI3ZTE2YzFiNDI4MGY0OTFmMGQ1YjM0ZjEyNzI1ZTBhOT
 BOT_LOGS_DIRECTORY='./bot_logs'
 MIN_BOT_VERSION='2.0.20'
 TIMEZONE=""
+TZ="UTC"
```

Comment:
is it necessary? As far as I know we have timestamp without timezone in the database.